appcelerator/appc-install
# [CLI-1203] Emojis should be removed from Appc CLI NPM

**Steps to verify:**
1. Run `npm install` in the repo
2. Run `./bin/appc use <CORE_VER>`
3. When the CLI starts downloading, immediately press `ctrl+c` to cancel the download

**Expected:** No emojis should appear.
